### ISSUE TITLE
Minor bug: iebaltab points to dmtab

### ISSUE THREAD

**Issue by de-barros** [worldbank/ietoolkit#137]
`iebaltab` points to `dmtab` [here](https://github.com/worldbank/ietoolkit/blob/4006f22aac0bad9c5b4eadaefe02bcab6a49e025/src/ado_files/iebaltab.ado#L496) and [here](https://github.com/worldbank/ietoolkit/blob/4006f22aac0bad9c5b4eadaefe02bcab6a49e025/src/ado_files/iebaltab.ado#L2228)

Usecase: 

![image](https://user-images.githubusercontent.com/20994418/40459101-8d27f7f6-5ecd-11e8-96b2-9ac3b8ad9c1d.png)

**Comment by kbjarkefur** [comment]
Thank you @de-barros!

Funny you found this bug as I found this bug myself two just weeks ago at the same time as this must have been there for a long time as `dmtab` was the name the command had before **ietoolkit** was a thing.

I fixed this in f5bed3f but it is not yet published as I found it after the last release and this is not a big enough bug to require a new release. Both those references are obviously meant to be to the help file of `iebaltab`. I am sure you figured this out yourself.

Nevertheless, thanks again for sending the issue despite a fix was on its way already!
